### DISCUSSION TITLE
Additional language in official docs

### DISCUSSION THREAD

**Discussion by worksofliam** [microsoft/vscode-discussions#224] [Extension Development QnA] [answered]
Hi! I maintain a language server for RPGLE, a language mostly written on an IBM i.

I don't really know who to ask, but how open are the VS Code doc maintainers to a page in the Language Support section on the official docs: https://code.visualstudio.com/docs/languages/overview#_common-questions

I built this page in our docs based on the existing TS/C++ pages, so it'd like look like this: https://halcyon-tech.github.io/docs/#/pages/extensions/rpgle/index

This would go a long way for me, the language and the RPGLE developer community.

**Comment by isidorn** [comment, marked as answer]
Thanks for reaching out. We only have language pages for languages that are broadly used by our users based on our telemetry information. RPGLE is currently not in the list of our top languages so for now we would not like to add it to the Language Support of the official docs. I hope that makes sense. 

fyi @gregvanl

**Comment by r3m0t** [comment]
@worksofliam you can get added to this page I think- https://microsoft.github.io/language-server-protocol/implementors/servers/
